atom/common/node_bindings.cc

```diff
@@ -239,8 +241,17 @@ void NodeBindings::UvRunOnce() {
   v8::MicrotasksScope script_scope(env->isolate(),
                                    v8::MicrotasksScope::kRunMicrotasks);
 
+  if (!is_browser_)
+    TRACE_EVENT_BEGIN1("devtools.timeline", "FunctionCall", "data",
```

Comment:
Looks like you can, I see the same results in the timeline with or without the empty data.